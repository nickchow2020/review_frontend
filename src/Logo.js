import React from "react";
import {ReactComponent as Logo} from "./thelogo.svg";
import "./Logo.css";

const MyLogo = ()=>{
    return(
        <>
            <Logo className="appLogo"/>
        </>
    )
}

export default MyLogo