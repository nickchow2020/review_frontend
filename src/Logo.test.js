import React from 'react';
import { render } from '@testing-library/react';
import Logo from "./Logo";


it ("should render", function(){
  render(<Logo />)
})


it ("should match snapshot", function(){
  const { asFragment } = render(<Logo />)

  expect(asFragment()).toMatchSnapshot()
})
