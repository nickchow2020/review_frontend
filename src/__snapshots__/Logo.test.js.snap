// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <svg
    class="appLogo"
  >
    thelogo.svg
  </svg>
</DocumentFragment>
`;
